src/Symfony/Bundle/FrameworkBundle/Resources/config/form.xml

```diff
@@ -204,5 +204,13 @@
             </property>
             <deprecated>The service "%service_id%" is internal and deprecated since Symfony 3.3 and will be removed in Symfony 4.0</deprecated>
         </service>
+
+        <!-- Commands -->
+        <service id="form.command.debug" class="Symfony\Component\Form\Command\DebugCommand">
```

Comment:
this service is invalid if the console is not installed, in that case we should remove this definition (see #23624)